packages/lame/lame.0.3.5/opam

```diff
@@ -0,0 +1,37 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "MP3 encoding library"
+description:
+  "Bindings for the lame library which provides functions for encoding mp3 files"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+license: "GPL-2.0"
+homepage: "https://github.com/savonet/ocaml-lame"
+bug-reports: "https://github.com/savonet/ocaml-lame/issues"
+depends: [
+  "conf-lame"
+  "dune" {> "2.0"}
+  "dune-configurator"
+]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
  ["dune" "subst"] {dev}
```

Reply:
Backported upstream & switched to `dune >= 2.8` thanks!